bloom_reachability_map.go

```diff
@@ -25,9 +29,8 @@ func (m BloomReachabilityMap) AddEntry(n NodeAddress) {
 	m.f.Add(entry)
 }
 
-// TODO: Figure out how to increment
 func (m BloomReachabilityMap) Increment() {
-	return
+	m.distance++
 }
 
 func (m BloomReachabilityMap) Merge(nr ReachabilityMap) error {
```

Comment:
Need to support distance merging being sane. Not just have everything be the same distance away.